Code/src/main/java/nl/utwente/group10/ui/components/lines/Connection.java

```diff
@@ -31,13 +37,13 @@ public Connection() {
     }
 
     public Connection(OutputAnchor from) {
-        this.setStartAnchor(from);
-        setEndPosition(from.getCenterInPane());
+        tryAddAnchor(from);
+        setEndPosition(getScenePoint(from));
```

Comment:
Ik denk dat dit een regressie is. Dit is wat er gebeurt met getScenePoint:

![regress](https://cloud.githubusercontent.com/assets/126832/7472353/e3dd68b6-f331-11e4-9330-7b733bdf5fa7.png)